### PR TITLE
add user manual cpu qos and best practice plugsched

### PR DESCRIPTION
**Pull request by songtao98**
This PR add user manual `CPU QoS` and best practice for Anolis Plugsched plug-in to support CentOS 7.9 for cpu qos feature.

### CODE REVIEW

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/best-practices/anolis_plugsched.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 2
+---
+
+# Anolis Plugsched
+
+针对 Koordinator 在 CentOS 7.9 操作系统上 CPU 混部能力的缺失，龙蜥社区开发人员给出了另一种方案，即利用 plugsched 调度器热升级技术提供一种 CPU 混部技术的调度器插件包。该插件可直接安装到 CentOS 7.9，不需要停机和业务迁移等工作。了解更多信息，请参阅[Blog](https://koordinator.sh/blog/anolis-CPU-Co-location)
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/best-practices/anolis_plugsched.md:7]: 为了提升xx操作系统内核在CPU资源维度的混部效果，龙蜥OS提供了一种插件化的解决方案

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/best-practices/anolis_plugsched.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  ```
+  # rpm -ivh https://github.com/koordinator-sh/koordinator/releases/download/v1.1.1/scheduler-bvt-noise-clean-$(uname -r).rpm --oldpackage
+  ```
+
+安装完成后，你可以在cpu cgroup目录下看到 `cpu.bvt_warp_ns` ，其使用方法与Group Identity特性兼容。
+
+### 移除插件
+
+移除插件可以使用 `rpm -e` 命令，然后 `cpu.bvt_warp_ns` 将也不再存在。请确保卸载插件前没有任何任务还在使用 `cpu.bvt_warp_ns` 。
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/best-practices/anolis_plugsched.md:33]: ## 使用Koordinator的CPU QoS功能
参考对应的用户文档

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/cpu-qos.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+## 简介
+
+Kubernetes支持将多种类型的应用以容器化的方式部署在同一台宿主机上运行，不同优先级的应用可能会竞争CPU资源，导致应用服务受损。Koordinator支持基于容器的QoS等级，优先保障高优先级应用的CPU性能。本文介绍如何使用容器CPU QoS功能。
+
+## 背景
+
+为了充分利用机器中的资源，通常会将高优先延迟敏感性LS（Latency-Sensitive）和低优先级BE（Best-Effort）的任务部署在同一台机器上，导致两种不同优先级任务之间存在资源竞争问题。Kubernetes根据应用的CPU Request/Limit，为容器设置物理资源限制，但仍存在容器间对CPU资源的竞争。例如，BE应用和LS应用共享物理核或逻辑核时，当BE应用负载较高时，会干扰LS应用的运行，导致服务响应延迟变高。
+
+为了提高LS应用使用CPU资源的稳定性，降低BE应用的干扰，Koordinator基于Alibaba Cloud Linux 2，提供了容器CPU QoS功能。Koordinator基于Group Identity提供的Linux调度优先级，差异化保障不同优先级应用的CPU调度，将LS应用标识为高优，BE应用标识为低优，在混合部署场景中有效改善LS应用的服务质量。更多信息，请参见[Group Identity功能说明](https://help.aliyun.com/document_detail/338407.htm#task-2129392)。
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/cpu-qos.md:11]: Koordinator基于Alibaba Cloud Linux和Anolis OS

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/cpu-qos.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+7. 执行以下命令，将be-pod-demo部署到集群。
+
+   ```undefined
+   kubectl apply -f be-pod-demo.yaml
+   ```
+
+8. 执行以下命令，在单机端的Cgroup分组中查看BE Pod的内核Group Identity生效情况。
+
+   ```python
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/cpu-qos.md:176]: some comments are not correct

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/cpu-qos.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+     name: ls-pod-demo
+     labels:
+       koordinator.sh/qosClass: 'LS' #指定Pod的QoS级别为LS。
+   spec:
+     containers:
+     - command:
+       - httpd
+       - -D
+       - FOREGROUND
+       image: registry.cn-zhangjiakou.aliyuncs.com/acs/apache-2-4-51-for-slo-test:v0.1
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/cpu-qos.md:109]: use the nginx image in koordintator repo

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**